google/cloud/bigtable/doc/bigtable-samples-instance-admin.dox

```diff
@@ -0,0 +1,65 @@
+/*!
+@page bigtable-samples-instance-admin Example: C++ Instance Admin Samples
+
+This example is a very simple "Instance Admin" application, written in C++, that
+illustrates how to:
+
+- Creates an instance(type: PRODUCTION) and run basic instance-operations.
```

Comment:
Maybe "Creates a production instance and runs basic instance admin operations."

Reply:
fixed.